### PR TITLE
Plackett copula

### PR DESCRIPTION
**Pull request by regislebrun**
This PR introduces a new copula, the only copula with constant odd ratio. It also provides the following features:
  *  Improved solvers: Now, Brent, Bisection and Secant can work on UniVariateFunction resulting into a huge speed improvement for cheap functions
  * Improved copulas: Now the parameter accessors are protected to avoid useless computation when the new value is equal to the current one
  * Improved conditional methods: Now it is based on UniVariateFunction classes for improved performances. For the Frank copula it is a 3x acceleration of the isoprobabilistic transformation.

### CODE REVIEW

File: python/src/PlackettCopula_doc.i.in [hunk 1]
```diff
@@ -0,0 +1,65 @@
+%feature("docstring") OT::PlackettCopula
+"Plackett copula.
+
+Available constructor:
```
Comment (jschueller) [inline python/src/PlackettCopula_doc.i.in]: no need to list available ctors when there's only one variant

File: python/src/PlackettCopulaFactory_doc.i.in [hunk 2]
```diff
@@ -0,0 +1,19 @@
+%feature("docstring") OT::PlackettCopulaFactory
+"Plackett Copula factory.
+
+Available constructor:
```
Comment (jschueller) [inline python/src/PlackettCopulaFactory_doc.i.in]: here too

File: python/doc/user_manual/probabilistic_modelling.rst [hunk 3]
```diff
@@ -214,6 +214,7 @@ Refer to :ref:`copula`.
     FrankCopula
     GumbelCopula
     IndependentCopula
+    PlackettCopula
```
Comment (jschueller) [inline python/doc/user_manual/probabilistic_modelling.rst]: I dont see PlackettCopulaFactory added somewhere in the rsts

**Comment by jschueller** [comment]
the changelog needs to be updated once #1300 is approved, also you can update the plackett headers (#1309 )

**Comment by jschueller** [comment]
If its going to 1.14, you can add it to the existing changelog, dont forget to list it in major changes

**Comment by jschueller** [comment]
Can you rebase to master ?

**Comment by jschueller** [comment]
It seems you had different fixes for the warning, now it doesnt build anymore :[

**Comment by regislebrun** [comment]
Ok to go?

**Pull request merged**